config/prod/ecmonsen-emorypipeline-infra.yaml

```diff
@@ -0,0 +1,22 @@
+template_path: "remote/ecmonsen-emorypipeline-infra.yaml"
+stack_name: "ecmonsen-emorypipeline-infra"
+dependencies:
+  - prod/ecmonsen-emorypipeline-ci.yaml
+parameters:
+  SourceAccountId: "186706593821"
+  SynapseStorageLocationId: "0"
+  SynapseNotifyUsers: "3399376"
+  Department: SysBio
+  Project: amp-ad
+  OwnerEmail: eva.monsen@sagebase.org
+hooks:
+  before_create:
+    # Note: After Sep 30, 2020 only "virtual-hosted" bucket URLs (beginning with https://<bucketname>.s3.amazonaws.com)
+    # are supported. See https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/
+    # Note: ecmonsen_emorypipeline_ci_code_bucket is the bucket created in the ecmonsen-emorypipeline-ci stack
+    # and its current value is in ../config.yaml
+
+    # The following command gets the lambda artifacts bucket name from the dependency stack (echoing it for logging)
+    - !cmd curl -sw "%{url_effective}\n" https://$(aws cloudformation describe-stacks --stack-name ecmonsen-emorypipeline-ci --query "Stacks[0].Outputs[?OutputKey == 'LambdaArtifactsBucketName'].OutputValue" --output text).s3.amazonaws.com/lambda-uploader/{{environment_variable.TRAVIS_BRANCH}}/ecmonsen-emorypipeline-infra.yaml --create-dirs -o templates/remote/ecmonsen-emorypipeline-infra.yaml
```

Comment:
Sorry, I just realized that the aws inside of this curl will not work for every repo because the role that's executing the command only has access to buckets in sandbox.  If we use this same command in the Sage-Bionetworks/scicomp-provisioner repo it will fail because the command will be run with a scicomp role.   I would just go with the static bucket name.

Reply:
Changed it to use a hard-coded URL. The URL will have to be different when this is deployed to scicomp.